Sources/TuistGenerator/Generator/ProjectGroups.swift

```diff
@@ -113,6 +115,12 @@ class ProjectGroups {
                              pbxproj: pbxproj)
     }
 
+    func buildMain() -> PBXGroup {
```

Comment:
Notice that every time we call this method we are running this sorting logic. That means that if we are not aware of this we might end up creating a performance bottleneck. What about making this explicit:

```swift
let projectGroups = ....
// Project groups where the groups have been sorted
let sortedProjectGroups = projectGroups.sorted()
```

Reply:
I've renamed `main` -> `sortedMain` to make it more explicit.